src/test/java/org/apache/maven/plugins/dependency/tree/TestTreeMojo.java

```diff
@@ -137,6 +146,93 @@ public void testTreeTGFSerializing() throws Exception {
         assertTrue(findString(contents, "testGroupId:release:jar:1.0:compile"));
     }
 
+    /**
+     * Test the JSON format serialization
+     */
+    public void testTreeJsonSerializing() throws Exception {
```

Comment:
Now that you have more robust JSON based tests I'd delete this test unless you care about JSON insignificant details like whether it's

"artifactId": "snapshot"

or

"artifactId"     :    "snapshot"